src/main_loader.js

```diff
@@ -55,12 +55,11 @@
   exports.UnexpectedResponseException = sharedUtil.UnexpectedResponseException;
   exports.OPS = sharedUtil.OPS;
   exports.UNSUPPORTED_FEATURES = sharedUtil.UNSUPPORTED_FEATURES;
-  exports.isValidUrl = sharedUtil.isValidUrl;
```

Comment:
Very good point, thanks for spotting this!
I've added a `isValidUrl` function as suggested in `display/dom_utils.js`.